build/devops_gated.yml

```diff
@@ -266,3 +266,28 @@ jobs:
       script: |
           cmake --build . | tee build.log
           if ((Select-String -Path "./build.log" -Pattern "Warning:") -ne $null) { throw "include check failed" }
+
+- job: includecheck_linux
+  pool:
+    name: Azure-MessagingStore
+    demands:
+      - linux
+
+  steps:
+
+  - bash: |
+     pushd $(Build.Repository.LocalPath)
+     git submodule update --init
```

Comment:
>git submodule update --init [](start = 5, length = 27)

Interesting, does this work for Linux? We had creds issues for Windows with it.
Maybe the machine has the credentials saved somehow :-)

Reply:
For Windows I added submodules:true

---
In reply to: [564146457](https://github.com/Azure/c-pal/pull/69#discussion_r564146457) [](ancestors = 564146457)

Reply:
copy-pasta from the normal Linux job above, so it works I guess?

---
In reply to: [564146540](https://github.com/Azure/c-pal/pull/69#discussion_r564146540) [](ancestors = 564146540,564146457)